arex-storage-web-api/src/main/java/com/arextest/storage/mock/impl/DefaultMockResultProviderImpl.java

```diff
@@ -31,50 +31,86 @@ final class DefaultMockResultProviderImpl implements MockResultProvider {
-                byte[] sequenceKey = sequencePut(key, valueRefKey);
-                if (sequenceKey != null) {
-                    size++;
-                }
+
+            for (T value : mockList) {
+                // Place the maximum number of playback times corresponding to the operations into the recorded data
+                byte[] recordOperationKey = CacheKeyUtils.buildRecordOperationKey(category, recordId, value.getOperationName());
+                int count = resultCount(recordOperationKey);
+                value.getTargetRequest().setAttribute(STREAMING_CALL_MAX, count);
+                size = sequencePutRecordData(category, recordIdBytes, size, recordKey, value);
             }
-            // if category type is the type to be compared.associate the mock instance id with the related mock key.
-            if (shouldUseIdOfInstanceToMockResult(category)) {
-                putRecordInstanceId(valueRefKey, value.getId());
-                putMockKeyListWithInstanceId(value.getId(), mockKeyList);
+        } else {
+            while (valueIterator.hasNext()) {
+                T value = valueIterator.next();
+                size = sequencePutRecordData(category, recordIdBytes, size, recordKey, value);
             }
         }
         LOGGER.info("put record result to cache size:{} for category:{},record id:{}", size, category, recordId);
         return size > EMPTY_SIZE;
     }
 
+    private <T extends Mocker> int sequencePutRecordData(MockCategoryType category, byte[] recordIdBytes, int size, byte[] recordKey, T value) {
+        List<byte[]> mockKeyList = matchKeyFactory.build(value);
+        final byte[] zstdValue = serializer.serialize(value);
+        byte[] valueRefKey = sequencePut(recordKey, zstdValue);
+        if (valueRefKey == null) {
+            return size;
+        }
+        for (int i = 0; i < mockKeyList.size(); i++) {
+            byte[] mockKeyBytes = mockKeyList.get(i);
+            byte[] key = CacheKeyUtils.buildRecordKey(category, recordIdBytes, mockKeyBytes);
+            byte[] sequenceKey = sequencePut(key, valueRefKey);
+            if (sequenceKey != null) {
+                size++;
+            }
+        }
+        // if category type is the type to be compared.associate the mock instance id with the related mock key.
+        if (shouldUseIdOfInstanceToMockResult(category)) {
+            putRecordInstanceId(valueRefKey, value.getId());
+            putMockKeyListWithInstanceId(value.getId(), mockKeyList);
+        }
+        return size;
+    }
+
     private boolean shouldUseIdOfInstanceToMockResult(MockCategoryType category) {
         return !category.isEntryPoint() && !category.isSkipComparison();
     }
 
+    private boolean shouldRecordStreamingCallMax(MockCategoryType category) {
+        return !category.isEntryPoint() && !category.isSkipComparison() && category.getName().startsWith(DUBBO_PREFIX);
```

Comment:
`return this.shouldUseIdOfInstanceToMockResult(category) && category.getName().startsWith(DUBBO_PREFIX);`